modules/porous_flow/src/dirackernels/PorousFlowLineGeometry.C

```diff
@@ -93,7 +94,7 @@ PorousFlowLineGeometry::PorousFlowLineGeometry(const InputParameters & parameter
                  "PorousFlowLineGeometry: wrong number of arguments - got ",
                  base.size(),
                  ", expected ",
-                 _mesh.dimension(),
+                 _mesh.dimension() + 1,
```

Comment:
This must not be tested? It seems like this would change behavior, where is this used?

Reply:
There is no test for this error message text.

Reply:
Just to be clear - the error message text was wrong before.  The only behavior changing here is just the printed message contents.